deltacat/storage/model/transaction.py

```diff
@@ -436,7 +444,47 @@ def _mark_end_time(self, time_provider: TransactionTimeProvider) -> int:
         end_time = self["end_time"] = time_provider.end_time()
         return end_time
 
-    def to_serializable(self) -> Transaction:
+    @staticmethod
+    def abs_to_relative(root: str, target: str) -> str:
```

Comment:
```suggestion
    def _abs_txn_meta_path_to_relative(root: str, target: str) -> str:
```
I think we can (1) make this method protected since only this class and its internal test cases should invoke it and (2) change its name to make it clear that it should only be used to relativize internal transaction metadata file paths (i.e., it has baked in assumptions about the shape of the root & target paths, and thus shouldn't be used to try to relativize any arbitrary root and target path).